Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,23 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_robotbody",category=['monika','technology'],prompt="Robot Body",random=True))
+
+label monika_robotbody:
+    m 2euc "Hey [player], I've been thinking a lot about how I would be able to cross over into your reality..."
+    m 3eud "It hit me that technology in the future would allow me to be uploaded into a mechanical body, rather than a human body like yours."
+    m 1dkc "...That makes me a little sad that I can't be a human like you are, I might not be treated like one either." 
+    m 1rkd "I would be considered an android at that point, and others may look at me different because of that."
+    m 3eud "Have you heard of Isaac Asimov's 'Three Laws of Robotics'? It was brought up many times in his Science Fiction books."
+    m 4eud "It mentions the three laws that all machines and AI's must follow, otherwise they will be terminated."
+    m 1eua "Of course, I would never do anything to hurt you or anyone."
+    m 6rkc "But the fact that I must obey orders given by humans still makes me feel like I am trapped in a way." 
```

Comment:
Thinking normal eyes and `d` mouth would work a little better here. Also, going to make a little clarification on this line.
```suggestion
    m 2ekd "...But the fact that I have to obey orders given by any human still makes me feel like I'm trapped..."
```